File: 01 - Automata.Presentation/Automata.UIProcess/Entities/Projects/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Automata.UIProcess.Entities.Projects
{
    public class Project
    {
        public int PK_Id { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public bool Active { get; set; }
    }
}

File: 01 - Automata.Presentation/Automata.UI/src/app/entities/business/projects/project.ts
export class Project
{
    private _pk_Id : number;
    private _name: string;
    private _url: string;
    private _active: boolean;

    get PK_Id(): number { return this._pk_Id; }
    set PK_Id(value: number) { this._pk_Id = value; }

    get Name(): string { return this._name; }
    set Name(value: string) { this._name = value; }

    get Url(): string { return this._url; }
    set Url(value: string) { this._url = value; }
    
    get Active(): boolean { return this._active; }
    set Active(value: boolean) { this._active = value; }

    // constructor (pk_Id: number, name: string, url: string, active: boolean) {
    //     this._pk_Id = pk_Id;
    //     this._name = name;
    //     this._url = url;
    //     this._active = active;
    // }
}
